charts/reform-scan-blob-router/values.yaml

```diff
@@ -33,7 +33,6 @@ java:
     STORAGE_URL: https://reformscan.{{ .Values.global.environment }}.platform.hmcts.net
     STORAGE_BULKSCAN_URL: https://bulkscan{{ .Values.global.environment }}.blob.core.windows.net
     STORAGE_BLOB_PROCESSING_DELAY_IN_MINUTES: 30
-    STORAGE_BLOB_SIGNATURE_ALGORITHM: "sha256withrsa"
```

Comment:
does that mean we need a separate PR just for config changes supplementing with chart version bump?

(which should then be done after this PR)

Reply:
bumped chart version. flux config pr: https://github.com/hmcts/cnp-flux-config/pull/1957